definitions/output/reports/cwv_tech_categories.js

```diff
@@ -8,43 +8,45 @@ publish('cwv_tech_categories', {
 /* {"dataform_trigger": "report_cwv_tech_complete", "name": "categories", "type": "dict"} */
 WITH pages AS (
   SELECT
-    root_page,
+    client,
+    root_page AS origin,
```

Comment:
A root_page has the trailing slash (e.g. `https://www.example.com/`) whereas an origin does not (`https://www.example.com`).

This is important because CrUX uses the term origin so I think we should be consistent with that.

So would prefer to keep `root_page` or drop the final slash.